styleguide/source/assets/scss/00-base/_webform_confirmation.scss

```diff
@@ -17,4 +17,11 @@
     display: flex;
     justify-content: center;
   }
+
+  &.ama__sales-landing-page__confirmation {
+    h1 {
```

Comment:
The confirmation `<h1>Thank You</h1>` is being hidden by `_sales-landing-page.scss`

Reply:
@tcfunk are your sure the thank you is being hidden? 
I initially hid the h1 because I didn't want the webform title to show
Then in the webform confirmation displayed it